app/presenters/organisations/show_presenter.rb

```diff
@@ -139,5 +163,17 @@ def needs_definite_article?(phrase)
     def has_definite_article?(phrase)
       phrase.downcase.strip[0..2] == 'the'
     end
+
+    def separate_job_links(corporate_information_links)
+      job_links = []
+
+      corporate_information_links.each do |link|
+        if link[:path].end_with?("/recruitment", "/procurement") || link[:text].eql?("Jobs")
```

Comment:
I can see why this has been done but it seems a bit precarious to separate the links like this. Not a blocking issue, but the publishing interface should let the publisher decide which list a link belongs to.